lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,64 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+        line_of_code: integer?,
+        last_commit_datetime: string
+      )
+    end
+
+    def analyzer_version
+      @analyzer_version ||= extract_version!("wc")
+    end
+
+    # This analyser use git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyze(changes)
+      Results::Success.new(guid: guid, analyzer: analyzer).tap do |result|
+        changes.changed_paths.each do |path|
+          result.add_issue(generate_issue(path, analyze_line_of_code(path), analyze_last_commit_datetime(path)))
+        end
+      end
+    end
+
+    private
+
+    def generate_issue(path, loc, commit_datetime)
+      Issue.new(
+        path: relative_path(path),
+        location: nil,
+        id: "metrics_fileinfo",
+        message: "#{path_str(path)}: loc = #{loc}, last commit datetime = #{commit_datetime}",
+        object: {
+          line_of_code: loc,
+          last_commit_datetime: commit_datetime
+        },
+        schema: Schema.issue
+      )
+    end
```

Comment:
[suggestion] How about passing only `path` variable to `generate_issue()` as follows?

This allows us to reduce the number of arguments of the `generate_issue()` method. Also the `path_str()` method is no longer needed.

```suggestion
    def analyze(changes)
      Results::Success.new(guid: guid, analyzer: analyzer).tap do |result|
        changes.changed_paths.each do |path|
          result.add_issue(generate_issue(path))
        end
      end
    end

    private

    def generate_issue(path)
      filepath = relative_path(path).to_s
      loc = analyze_line_of_code(filepath)
      commit_datetime = analyze_last_commit_datetime(filepath)

      Issue.new(
        path: relative_path(path),
        location: nil,
        id: "metrics_fileinfo",
        message: "#{filepath}: loc = #{loc}, last commit datetime = #{commit_datetime}",
        object: {
          line_of_code: loc,
          last_commit_datetime: commit_datetime
        },
        schema: Schema.issue
      )
    end
```

Reply:
It's so smart. I've fixed in fda89f5c2c1be2b7775e82bd2f9afcfc90a5d995.

Reply:
There is `def path_str: (Pathname) -> String` remaining in [metrics_fileinfo.rbs](https://github.com/sider/runners/blob/fda89f5c2c1be2b7775e82bd2f9afcfc90a5d995/sig/runners/processor/metrics_fileinfo.rbs#L19). Thank you.

Reply:
Thanks cdee994cde7d8d9e6c4424b27741209de85e4fc2